haskell-overlays/untriaged.nix

```diff
@@ -48,6 +48,11 @@ in self: super: {
   witherable = self.callHackage "witherable" "0.3.1" {};
   time-compat = dontCheck super.time-compat;
   bimap = self.callHackage "bimap" "0.3.3" {};
+  reflex-vty = self.callHackageDirect
+    { pkg = "reflex-vty";
+      ver = "0.1.2.1";
+      sha256 = "1qnnf76ddz8z27lgly7s6clw3lcljahgydhz7a2pqzmmarqq4zm8";
+    } {};
```

Comment:
nit: This indentation style is not idiomatic nix

Reply:
what is idiomatic nix?

Reply:
```nix
  reflex-vty = self.callHackageDirect {
    pkg = "reflex-vty";
    ver = "0.1.2.1";
    sha256 = "1qnnf76ddz8z27lgly7s6clw3lcljahgydhz7a2pqzmmarqq4zm8";
  } {};
```

Reply:
"idiomatic" is an annoying word; so I'll just say that looking at your original I know you write Haskell. But I can't recall seeing that in most nix I've read. :smile:

Reply:
fixed, i think